pkg/controller/framework/informers/factory.go

```diff
@@ -82,3 +82,18 @@ func CreateSharedNodeIndexInformer(client clientset.Interface, resyncPeriod time
 
 	return sharedIndexInformer
 }
+
+func CreateSharedNamespaceInformer(client clientset.Interface, resyncPeriod time.Duration) framework.SharedInformer {
```

Comment:
Go ahead and have this return the IndexInformer flavor.  I don't think there's a reason to construct the "normal" one.